debian/patches/0001-feat-radxa-common-kernel-config.patch

```diff
@@ -929,6 +929,10 @@ index 000000000000..c26829dd1cae
 +# Prefer rtw89-dkms
 +CONFIG_RTW89=n
 +CONFIG_RTW89_8852BE=n
--- 
++
++# Enable optional virtual filesystems
```

Comment:
```suggestion
+# Enable additional language support
```

Reply:
Please also move your change before `# Prefer r8125-dkms` section.

Reply:
Alright, did so.

Just noting `Enable optional virtual filesystems` was the section of the auto-generated configuration these two options normally appear with.